lib/tls.js

```diff
@@ -236,7 +236,6 @@ exports.parseCertString = internalUtil.deprecate(
   'Please use querystring.parse() instead.',
   'DEP0076');
 
-// Public API
```

Comment:
why remove this?

Reply:
Its a misleading comment. These aren't the only exported functions.